### DISCUSSION TITLE
How to monitor core's read or write access information to pmm？

### DISCUSSION THREAD

**Discussion by Yxiaoge** [opcm/pcm#277] [Q&A] [answered]
Remote/local DRAM access displayed in pcm-numa is at core granularity. 
1.  What should I do if I want to observe the access status at thread granularity? Could PCM monitor threads by thread_id as in perf?
2. What should I do if I want to observe the read/write access of each core to pmm

**Comment by opcm** [comment, marked as answer]
1. PCM does not support OS thread granularity.
2.  Similar to [the other question](https://github.com/opcm/pcm/discussions/260)  you can collect OFFCORE_RESPONSE PMM events for reads and writes (RFOs). I know they have gaps but a subset of them should give you an idea of the read/write distribution.
